contributors/guide/contributor-cheatsheet/README-it.md

```diff
@@ -0,0 +1,395 @@
+piattaforma, [Prow]. Se uno dei test fallisce, il [k8s-ci-robot] [prow]
+risponderà al PR con link ai test falliti e ai log disponibili.
+
+Spingendo nuovi commit al tuo PR, i test verranno automaticamente riavviati.
+
+Occasionalmente possono esserci problemi con la piattaforma CI di Kubernetes. Questi possono accadere
+per una vasta gamma di motivi, anche se il tuo contributo passa a livello locale
+test. È possibile attivare una nuova esecuzione dei test con il comando `/ retest`.
+
+Per ulteriori informazioni sulla risoluzione di test specifici, consultare la [Guida al test].
+
+
+### Etichette
+
+Kubernetes usa [etichette] per categorizzare e triage problemi e Pull Requests.
+Applicare le etichette giuste aiuterà di più il tuo problema o le tue PR
+in modo efficace.
+
+**Riferimenti:**
+- [Etichette]
+- [Prow commands] [comandi]
+
+Etichette usate frequentemente:
+- [`/ sig <sig name>`] [tipo] Assegna un [SIG] [SIGs] alla proprietà del problema
+  o PR.
+- [`/ area <nome area>`] [tipo] Associa il problema o le PR a uno specifico
+  [AREA] [] etichette.
+- [`/ kind <category>`] [kind] [Categorizes] [labels] il problema o PR.
+
+---
+
+## Lavorare localmente
+
+Prima di proporre una richiesta di pull, dovrai svolgere un certo livello di lavoro
+localmente. Se sei nuovo a git, il [tutorial git Atlassian] è un buon inizio
+punto. In alternativa, il tutorial di [Git magic] di Stanford è un bene
+opzione multi-lingua.
+
+**Riferimenti:**
+- [Atlitian git tutorial]
+- [Git magic]
+- [flusso di lavoro Github]
+- [Testing localmente]
+- [Guida per gli sviluppatori]
+
+
+### Strategia del ramo
+
+Il progetto Kubernetes utilizza un flusso di lavoro _ "Fork and Pull" _ standard
+GitHub. In termini git, la tua forcella personale viene chiamata _ "` origine` "_ e
+il vero repository git del progetto è chiamato _ "` upstream` "_. Per mantenere il tuo
+ramo personale (`origine`) aggiornato con il progetto (` upstream`), deve essere
+configurato all'interno della tua copia di lavoro locale.
+
+
+#### Aggiunta di Upstream
+
+Aggiungi `upstream` come remoto e configuralo in modo da non poterlo spingere.
+
+`` `
+# sostituisce <upstream git repo> con l'url repo upstream
+# esempio:
+# https://github.com/kubernetes/kubernetes.git
+# git@github.com/kubernetes/kubernetes.git
+
+git remote add upstream <upstream git repo>
+git remote set-url --push upstream no_push
+`` `
+
+Questo può essere verificato eseguendo `git remote -v` che elencherà i tuoi configurati
+telecomandi.
+
+
+#### Mantenere la tua forcella in sincronizzazione
+
+Recupera tutte le modifiche da `upstream` e _" rebase "_ sul tuo` master` locale
+ramo. Questo sincronizzerà il repository locale con il progetto `upstream`.
+
+`` `
+git fetch upstream
+git checkout master
+git rebase upstream / master
+`` `
+
+Dovresti fare questo minimamente prima di creare un nuovo ramo su cui lavorare
+funzionalità o correzione.
+
+`` `
+git checkout -b myfeature
+`` `
+
+#### Impegni di schiacciamento
```

Comment:
Completely wrong. To english this would translate to "Crushing Duties", it doesn't make any sense.